jenkins/common/sonic-mgmt-framework-build-pr/Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+pipeline {
+    agent { node { label 'sonic-slave' } }
+
+    stages {
+        stage('Prepare') {
+            steps {
+                dir('sonic-mgmt-framework') {
+                    checkout([$class: 'GitSCM',
+                          branches: [[name: '${sha1}']],
+                          userRemoteConfigs: [[url: 'http://github.com/Azure/sonic-mgmt-framework',
+                                               refspec: '+refs/pull/*:refs/remotes/origin/pr/*']]])
+                }
+                copyArtifacts(projectName: 'vs/buildimage-vs-all', filter: '**/*.deb', target: 'buildimage', flatten: false)
```

Comment:
>vs [](start = 44, length = 2)

Do you mean `../vs` ?

Reply:
I am not sure how the path is picking up. Can you please help clarify.

If u see below pr jobs they have same path config as above. 

**lvnvde1934#cat sonic-swss-common-build-pr/Jenkinsfile | grep copyArt
                copyArtifacts(projectName: 'vs/buildimage-vs-all', filter: '**/*.deb', target: 'buildimage', flatten: false)

lvnvde1934#cat sonic-telemetry-build-pr/Jenkinsfile | grep copyAr
                copyArtifacts(projectName: 'vs/buildimage-vs-all', filter: '**/*.deb', target: 'buildimage', flatten: false)**


Only build jobs have ../vs path. 

lvnvde1934#cat sonic-swss-common-build/Jenkinsfile | grep copyArt
                copyArtifacts(projectName: '../vs/buildimage-vs-all', filter: '**/*.deb', target: 'buildimage', flatten: false)